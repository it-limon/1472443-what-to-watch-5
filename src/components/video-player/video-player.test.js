import React, {createRef} from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player";

const ref = createRef();

describe(`Render VideoPlayer`, () => {

  it(`Render VideoPlayer muted`, () => {
    const tree = renderer
      .create(
          <VideoPlayer
            videoRef={ref}
            src={`src.avi`}
            poster={`poster.jpg`}
            isMuted={true}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Render VideoPlayer not muted`, () => {
    const tree = renderer
      .create(
          <VideoPlayer
            videoRef={ref}
            src={`src.avi`}
            poster={`poster.jpg`}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
