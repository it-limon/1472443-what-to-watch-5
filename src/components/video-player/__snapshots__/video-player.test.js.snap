// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render VideoPlayer Render VideoPlayer muted 1`] = `
<video
  className="player__video"
  muted={true}
  poster="poster.jpg"
  src="src.avi"
/>
`;

exports[`Render VideoPlayer Render VideoPlayer not muted 1`] = `
<video
  className="player__video"
  muted={false}
  poster="poster.jpg"
  src="src.avi"
/>
`;
